Website/js/Debug/dnn.modalpopup.js

```diff
@@ -206,13 +204,12 @@
 
                         var $window = $(window),
                             newHeight,
-                            newWidth,
-                            JQUERY_UI_HEIGHT_SHRINK_OFFSET = 100,
+                            newWidth,                            
                             horizontalPosition = "center",
                             verticalPosition = "center";                            
 
                         if ($modal.data('isMaximized')) {
-                            newHeight = $modal.data('height') + JQUERY_UI_HEIGHT_SHRINK_OFFSET;
```

Comment:
I'm not sure why this line get removed, its add in DNN-8295, if this line really need to removed, we should ask QA to verify DNN-8295 won't broken with the changes as well.